man/rgbasm.1

```diff
@@ -130,6 +131,13 @@ Write an object file to the given filename.
 .It Fl p Ar pad_value , Fl Fl pad-value Ar pad_value
 When padding an image, pad with this value.
 The default is 0x00.
+.It Fl Q Ar fix_precision , Fl Fl q-precision Ar fix_precision
+Use this as the precision of fixed-point numbers after the decimal point.
+The default is 16, so fixed-point numbers are Q16.16.
+The argument may start with a
+.Ql .
+to match the Q notation, for example,
+.Ql Fl Q Ar .16 .
```

Comment:
My knowledge with c and rgbds' inner workings is limited; however from a cursory review, with my limited experience, this looks good to me. Just one question/nitpick.

I'm assuming this just overrides the _**default**_ precision value of `16`; and you can still specify specific instances of different precision. If this is the case, can we slightly change the wording here to make that fact more obvious?

Reply:
Yes, the setting from `-Q` or `opt Q` applies to fixed-point literals (and to fixed-point functions), but not to explicit-precision literals like `12.34q8`. Those should be described in the docs.